docs/Developers/Newcomers_Guide.rst

```diff
@@ -164,6 +164,34 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
 We also have a `page dedicated to git commands <http://coala.io/git>`_ that
 will help you learn the basics.
 
+.. note::
+
+    coala also has some of its repositories on
+    `Gitlab <https://gitlab.com/coala>`_.
+    Gitlab contributions also count for the promotion of
+    newcomer to developer process.
+
+Why Gitlab?
```

Comment:
why this section is created? I don't see the issue mentions anything about explaining the importance of gitlab